example/src/screens/Home.tsx

```diff
@@ -1,17 +1,18 @@
-import React, {useEffect, useState} from 'react';
-import {Platform, Button, Text, View} from 'react-native';
-import PointSdkRn, {FitbitScopes, OuraScopes} from 'react-native-point-sdk';
+import React, { useEffect, useState } from 'react';
+import { Platform, Button, Text, View } from 'react-native';
+import PointSdkRn, { FitbitScopes, OuraScopes } from 'react-native-point-sdk';
+import Config from "react-native-config";
 
 export function HomeScreen() {
   const [user, setUser] = useState<PointSdkRn.User | null>(null);
   const [ouraStatus, setOuraStatus] = useState<string>('unknown');
   const [fitbitStatus, setFitbitStatus] = useState<string>('unknown');
 
-  useEffect(() => {
-    handleLogin();
-    isOuraAuthenticated();
-    isFitbitAuthenticated();
-  }, []);
+  // useEffect(() => {
```

Comment:
Commented this out because this is causing a setToken call before the SDK setup, therefore causing errors (and crash)

Reply:
You can probably remove this piece of code instead of commenting it out.